components/admin_console/admin_sidebar/admin_sidebar.jsx

```diff
@@ -147,16 +147,29 @@ export default class AdminSidebar extends React.Component {
                 result.add(item.url);
             }
         }
-        for (const item of Object.values(AdminDefinition.other)) {
+        for (const item of Object.values(AdminDefinition.user_management)) {
             if (isVisible(item)) {
                 result.add(item.url);
             }
         }
-        for (const section of Object.values(AdminDefinition.settings)) {
-            for (const item of Object.values(section)) {
-                if (isVisible(item)) {
-                    result.add(section.url + '/' + item.url);
-                }
+        for (const item of Object.values(AdminDefinition.environment)) {
```

Comment:
I think we can solve it with a loop inside another loop, right?

Reply:
Yes, I'm actually refactoring a bit right now. Was waiting until content seemed solid.

Reply:
So, maybe wait to review after a few hours : )